index.d.ts

```diff
@@ -20,6 +20,11 @@ export type ThrottledFunction<Argument extends readonly unknown[], ReturnValue>
 		Abort pending executions. All unresolved promises are rejected with a `pThrottle.AbortError` error.
 		*/
 	abort(): void;
+
+	/**
+		The number of queued items waiting to be executed.
+		*/
```

Comment:
Incorrect indentation. Always look over your own diff.

Reply:
I did look over the diff, and made sure to match the indentation of existing comments in the type!

What I've added here is consistent with the indentation in other comments on the type `ThrottledFunction` (see above my change), but type `ThrottledFunction` has different indentation to interface `Options` (see below my change).

Would you like me to do one of the following?

• fully change the indentation on comments in type `ThrottledFunction` to match that of interface `Options`,
• exclusively change the line I'm adding to be consistent with the comment indentation in interface `Options`, leaving the other comments on the type alone, or 
• leave as-is